docs/README.md

```
@@ -266,6 +266,30 @@ Now you can see all those in a specific new type category in your backend (proba
 This is auto-detected based on [Search](https://github.com/FriendsOfCake/search) plugin being available/loaded if not disabled.
 If disabled, it will at least still be able to filter by error type.
 
+## Auto-creation of the logs table
+
+For convenience the plugin can create its `database_logs` table on the fly, the first time the
+table class is instantiated. **This happens in `debug` mode only.** In production you are expected
+to have run the migration:
+
+    bin/cake migrations migrate -p DatabaseLog
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)